query-parser/src/parse.rs

```diff
@@ -219,18 +225,20 @@ def_value_parser_fn!(Find, find, FindSpec, input, {
 //
 #[allow(dead_code)]
 pub fn find_seq_to_find_spec(find: &[edn::Value]) -> FindParseResult {
-    Find::find()
-        .parse(find)
-        .map(|x| x.0)
-        .map_err(|_| FindParseError::Err)
```

Comment:
@ncalexan: pretty sure this is familiar to you. Combine's `ParseError` doesn't implement `Error`, so we can't use `chain_err`. `Result` isn't `Result`, and the compiler can't figure out the types, so we can't just use `map_err` here. (I even tried `.map_err(…).chain_err()`, and it complained about that, too.)

We also can't directly add a `ParseError` as an `ErrorKind`, because it has an associated lifetime and is generic up the wazoo, as before. I believe you have code to improve this, so we can align after that lands!